docs/others/obj_id.rst

```diff
@@ -2,27 +2,27 @@
 OBJ ID
 ======
 
-LVGL provides an optional field in `lv_obj_t` to store the object ID.
+LVGL provides an optional field in :cpp:type:`lv_obj_t`` to store the object ID.
 Object ID can be used in many cases, for example, to identify the object.
 Or we can store a program backtrace to where the object is created.
 
 Usage
 -----
 
-Enable this feature by setting ``LV_USE_OBJ_ID`` to 1 in `lv_conf.h`.
-Use the builtin obj ID generator by setting ``LV_USE_OBJ_ID_BUILTIN`` to 1.
+Enable this feature by setting :c:macro:`LV_USE_OBJ_ID` to `1`` in ``lv_conf.h``.
+Use the builtin obj ID generator by setting :c:macro:`LV_USE_OBJ_ID_BUILTIN` to `1`.
 Otherwise provide your own custom implementation.
 
-The ID is automatically generated and assigned to `obj->id` during obj's
-construction by calling API `lv_obj_assign_id(obj)` from ``lv_obj_constructor``.
+The ID is automatically generated and assigned to :cpp:expr:`obj->id`: during obj's
```

Comment:
:cpp:expr:\`obj->id\`

additional colon ":" at the end of the expression

Reply:
removed